cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -615,7 +615,7 @@ features:
         uploadSubTitle: Select the file to be uploaded to the workspace
```

Comment:
DataBrowser -> No Plugin is no longer used. Could we remove that. It was a temporary solution.